.changeset/stale-games-shake.md

```diff
@@ -0,0 +1,5 @@
+---
+"@sveltejs/kit": patch
+---
+
+Fix: treated shallow are as such
```

Comment:
```suggestion
fix: treat shallow `pushState` navigation as a navigation
```

Reply:
Oh, i saw you changed it to
fix: `popstate` navigations take `pushState` navigations into account

That makes sense :+1: